memtable.go

```diff
@@ -207,6 +207,7 @@ func (mt *memTable) Put(key []byte, value y.ValueStruct) error {
 	if ts := y.ParseTs(entry.Key); ts > mt.maxVersion {
 		mt.maxVersion = ts
 	}
+	y.NumBytesLSMWrittenAdd(mt.opt.MetricsEnabled, entry.estimateSizeAndSetThreshold(mt.opt.ValueThreshold))
```

Comment:
At first, I thought this was total LSM writes to disk (including aplification from compactions) but now I think it is the amount of data flushed to L0 total (writes without amplification). Can we name this more specifically, perhaps NumBytesLSMWrittenPreAmplify, or NumBytesFlushedToL0 ?